ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -49,100 +40,72 @@ def hello_endpoint(helloid):
         self.client = Client(self.app, BaseResponse)
 
     def test_simple(self):
+        expected_attrs = {
```

Comment:
some of these expected_attrs looks pretty common. could they be pooled into some common constructor or factory?

Reply:
updated to use a method with a dict argument for anything that the test needs to override